doc/site/content/changelogs/changelog-2025-03.markdown

```diff
@@ -2,27 +2,30 @@
 title = "Release 2025.03"
```

Comment:
Maybe rename the filename and title?

Reply:
hm.. I'm concerned here, I'd rather have the filename accurate. Would you mind if I check and assure you:

- We have `2025-03` available so links don't break and point to `2025-04`?
- I fix any references to `2025-03` in site to `2025-04`?

If you do mind, then feel free to resolve this.

Reply:
Sure, renamed.

I'd be happy if existing links to 03 keep working, but it's more about the principle and having some awareness/experience for handling this kind of rename in the future, rather than an expectation that such links actually exist in the wild and could see use.

Reply:
Yeah, I agree with you. See if the redirect I suggested [here](https://github.com/beyond-all-reason/RecoilEngine/pull/2365/files#r2267461820) makes sense.

See: https://gohugo.io/content-management/urls/#alias-examples